pulpcore/cli/common/generic.py

```diff
@@ -233,6 +241,65 @@ def repair(
     return callback
 
 
+def label_command(**kwargs: Any) -> click.Command:
+    """A factory that creates a label command group."""
+
+    if "name" not in kwargs:
+        kwargs["name"] = "label"
+    decorators = kwargs.pop("decorators", [])
+    lookup_decorators = kwargs.pop("lookup_decorators", [name_option, href_option])
+
+    @click.group(**kwargs)
+    @pass_entity_context
+    @pass_pulp_context
+    @click.pass_context
+    def label_group(
+        ctx: click.Context, entity_ctx: PulpEntityContext, pulp_ctx: PulpContext
+    ) -> None:
+        pass
+
+    for option in decorators:
+        # Decorate callback
+        label_group = option(label_group)
```

Comment:
The only decorators i'd expect to see here are the entity lookup ones if we go for interspersed args. And yet they can be attached one level higher.
I'd prefer to pass the lookup options explicitly as the decorator argument.